### ISSUE TITLE
Polyp Segmentation in Endoscopy Image

### ISSUE THREAD

**Issue by Kaushal-11** [abhisheks008/DL-Simplified#626]
I propose to add a feature for Polyp Segmentation using an endoscopy image dataset. This feature aims to improve the detection and segmentation of polyps in endoscopic images, which is crucial for early diagnosis and treatment of colorectal cancer.

Kindly assign me the issue under the label of GSSoC'24.

Dataset  : it contains various images by taking endoscoy.
Link : https://polyp.grand-challenge.org/CVCClinicDB/

Model : U - Net model is used for segmentation with using tensorflow library.

**Comment by abhisheks008** [comment]
Hi @Kaushal-11 can you implement at least 3 models for this project?

**Comment by Kaushal-11** [comment]
Yes, i can implement different models also like DeepLab, FCN, ResNet.

**Comment by abhisheks008** [comment]
Cool, issue assigned to you @Kaushal-11

**Comment by Kaushal-11** [comment]
Hey, @abhisheks008 
UNet model's training accuracy has reached 0.96, and the validation accuracy is at 0.93

**Comment by abhisheks008** [comment]
> Hey, @abhisheks008 UNet model's training accuracy has reached 0.96, and the validation accuracy is at 0.93

What about the rest of the models?

**Comment by Kaushal-11** [comment]
I am working on it

**Comment by Kaushal-11** [comment]
Hey, @abhisheks008 
2nd Model ResUnet model's training accuracy has reached 0.98, and the validation accuracy is at 0.95

And 3rd model is under working

**Comment by Kaushal-11** [comment]
Hey, @abhisheks008 
I have completed my work on all models.

ResUnet model give accuracy 0.95,
DeepLabV3+ model give accuracy 0.94

**Comment by Kaushal-11** [comment]
I have done my task with 3 model,

so can you suggest me how to add this stuff in repository ?

**Comment by abhisheks008** [comment]
> I have done my task with 3 model,
> 
> so can you suggest me how to add this stuff in repository ?

Clone/fork this repo, make the changes in the forked repo and push your code by a PR.

**Comment by Kaushal-11** [comment]
okay, I have done it.
Now check all the stuffs, 
If you see any areas that need improvement please inform me.

**Comment by Kaushal-11** [comment]
@abhisheks008 

Hey @SrijanShovit,

Thank you for your guidance throughout this project.
